addons/block-palette-icons/addon.json

```diff
@@ -1,14 +1,17 @@
 {
   "name": "Block palette category icons",
-  "description": "Replaces the colored circles that identify block categories with custom icons.",
-  "tags": ["editor"],
+  "description": "Adds icons inside the colored circles that normally identify block categories.",
```

Comment:
Explain why the description change?

Reply:
just for clarity. the addon doesn't remove the circles, they're still there. it just adds icons inside them.

Reply:
Ok, but can you remove `normally`? Not sure why you added that keyword